farm/modeling/language_model.py

```diff
@@ -103,6 +103,18 @@ def load(cls, pretrained_model_name_or_path, **kwargs):
                 f"models: https://farm.deepset.ai/api/modeling.html#farm.modeling.language_model.LanguageModel.load"
             )
 
+        # resize embeddings in case of custom vocab
+        if n_added_tokens != 0:
+            # TODO verify for other models than BERT
+            model_emb_size = language_model.model.resize_token_embeddings(new_num_tokens=None).num_embeddings
+            vocab_size = model_emb_size + n_added_tokens
+            logger.info(
+                f"Tokenizer vocabulary (len: {vocab_size}) does not match original language model vocabulary (len: {model_emb_size}). Resizing embedding layer of LM accordingly")
```

Comment:
It would be good to rephrase this message if possible. We want to differentiate cases where the vocab size mismatch arises due to wrong components vs the introduction of added vocab